lib/errors/error_handling.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module ErrorHandling
+  def log_error(error_record)
+    Rails.logger.info "Caught #{error_record.error}"
+    error_record.level = error_level(error_record.error)
+    if error_record.specific_error_logging?
```

Comment:
I really don't like the level of indirection happening with this module vs ErrorRecord vs the update object.

I'm not convinced that `ErrorRecord` gets us anything. I think the ideal situation is that with this module, there's basically just a single method that each API class needs to know (`log_error`). If we need to create an `ErrorRecord` (which I'm still open to), that could be one inside this method, instead of in the API-specific code that gets duplicated across API files.

Reply:
Now that the number of arguments are only three, `error`, `sentry_extra` and `update_service`, they can be passed into this method easily. After passing the parameters, I guess `ErrorRecord` won't be of as much help as it was earlier, should I remove it or not?